test/component/setup_pubsub.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+echo 'Running setup_pubsub.sh'
+
+wait_for_curl_success() {
+    local healthcheck_url=${1}
+    local http_verb=$2
```

Comment:
Should this also be in curly brackets?

```suggestion
    local http_verb=${2}
```

Reply:
yes, though it seemed to work without it, which is interesting, for a given value of interesting